packages/data-transport-layer/src/services/l1-ingestion/service.ts

```diff
@@ -114,6 +115,10 @@ export class L1IngestionService extends BaseService<L1IngestionServiceOptions> {
       this.options.addressManager
     )
 
```

Comment:
This now adds an explicit dependency on the L2 chain id being available on L1 - we have spoken about not deploying the execution manager on L1 until a fraud proof is being executed, so this is not safe under that assumption. This will have to pull from the environment manager in the future